_posts/2023-11-23-announcing-dp-social-inbox.md

```diff
@@ -0,0 +1,101 @@
+
+All of Sutty's templates are compatible with the Social Inbox, in addition to the Magazine template, which is based on the layout of COMPOST magazine, our sister project. For a step-by-step guide on how to use the Social Inbox with Sutty, check out this [LINK:demo video].
+
+[LINK WHEN AVAILABLE: You can see how it looks by checking out our account on the Fediverse: @HyphaCoop@hypha.coop]
+
+## Getting Started
+
+There are a few ways to try out the Social Inbox:
+
+* **Sutty**: Create a static website on Sutty and use the Social Inbox panel on their platform admin panel: https://sutty.nl/en/
+
+* **Do it yourself**: Check out our documentation to see how you can set up the Social Inbox on your existing (or future) static website: https://docs.distributed.press/
+
+* **Hire us to help**: If you already have a static website publishing pipeline, but want to publish to IPFS and Hypercore and implement the Social Inbox, get in touch with us at hello@distributed.press and we can help you out. :)
+
+## What's Next
+
+Our next area of focus is to make it easier for people to discover websites and blogs on the dweb easier. Essentially, we're developing automation on top of Distributed Press that will announce newly published accounts in a format that can be subscribed to on the Fediverse. 
+
+We also see the Social Inbox, but particular the Inbox Server, as a stepping stone towards further decentralization. We hope to make publishing and social media local-first and peer-to-peer by creating paths to data storage that puts the data closer to the end-user. For example, all the data related to your website and its social interactions could be hosted on your own computer, or another local server of your choosing. One of our next steps in this direction is going to be able to create a p2p-enabled Fediverse client and a specification for loading ActivityPub content from p2p protocols.
+
+Also, if you're a developer on another static site generator (that isn't Jekyll), [please reach out](mailto:hello@distributed.press) if you're interested in building a similar plug-in to integrate ActivityPub into your publishing workflow.
+
+Stay tuned and thanks for reading! 
+
+## Support Us!
+
+Love what we do and want to help sustain this project? **[You can support us on Open Collective with a one-time gift or monthly donation.](https://opencollective.com/compost) **
+
+*~*~*~*~*
+
+### About this project
+Distributed.Press is a joint collaboration between two tech worker co-operatives, Hypha Worker Co-operative based in Toronto, Canada and Sutty based in Buenos Aires, Argentina.
+
+The project is made possible through generous support from [Filecoin Foundation for the Decentralized Web](https://ffdweb.org/), Grant for the Web, and our supporters on [Open Collective](https://opencollective.com/compost). 
```

Comment:
I think this should be the DP collective link, right? https://opencollective.com/distributed-press

Reply:
Yes!